CHANGES.md

```diff
@@ -4,6 +4,8 @@ Changes
 development (master)
 --------------------
 
+- Add system-wide `.../name/name.yaml` paths to the default load order, aiding in the use configuration *directories* (e.g. in containerized setups). 
```

Comment:
Do I understand it correctly that the primary use case here is mounting an entire config (with data) directory in a container? Perhaps instead of mounting just a single yaml file?

Reply:
Never mind. I just saw the description of this PR. Question answered 👌🏻. Doing this on my phone, so I missed it initially.

Reply:
Well, at least that makes things line up with the expectations :D